prow/manifests/overlays/metal3/config.yaml

```diff
@@ -771,71 +723,7 @@ presubmits:
                 value: "TRUE"
             image: docker.io/golang:1.19
             imagePullPolicy: Always
-    - name: gofmt
-      branches:
-        - main
-      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
-      decorate: true
-      spec:
-        containers:
-          - args:
-              - ./hack/gofmt.sh
-            command:
-              - sh
-            env:
-              - name: IS_CONTAINER
-                value: "TRUE"
-            image: docker.io/golang:1.21
-            imagePullPolicy: Always
-    - name: gofmt
-      branches:
-        - release-1.5
-        - release-1.6
-      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
-      decorate: true
-      spec:
-        containers:
-          - args:
-              - ./hack/gofmt.sh
-            command:
-              - sh
-            env:
-              - name: IS_CONTAINER
-                value: "TRUE"
-            image: docker.io/golang:1.20
-            imagePullPolicy: Always
-    - name: gofmt
-      branches:
-        - release-1.4
```

Comment:
1.4 removed?

Reply:
Now I have confused myself. :dizzy_face: 
I see that we have [golangci-lint here](https://github.com/metal3-io/project-infra/blob/1b7e776fbdafd4845ab3ce51a49c14fabb42566b/prow/manifests/overlays/metal3/config.yaml#L880-L895) for CAPM3 release-1.4. That calls [this](https://github.com/metal3-io/cluster-api-provider-metal3/blob/release-1.4/hack/ensure-golangci-lint.sh) which runs [golangci-lint](https://github.com/metal3-io/cluster-api-provider-metal3/blob/7d289d9ac9aa4d928ac4bbd8cf0b4eef92bd3cac/Makefile#L265-L269).

So it seems like we have golangci-lint here but not the github workflow that we use for newer branches. :thinking: 
Go we still need the gofmt here then?

Reply:
1.4's `.golangci.yaml` has gofmt enabled, and the linter is called individually per root, api, and test, so it should work. I think it can be dropped.

Reply:
safe to remove